pkgs/development/python-modules/haralyzer/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  python-dateutil,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "haralyzer";
+  version = "2.4.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "haralyzer";
+    repo = "haralyzer";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-4Q0kNorZwE1zvh768sLRgIP4vRvI1Ide3uSfzqL5oIU=";
+  };
+
+  build-system = with python3Packages; [
```

It's a Python module.

```suggestion
  build-system = [
```